config/core/webhooks/defaulting.yaml

```diff
@@ -28,3 +28,32 @@ webhooks:
   sideEffects: None
   name: webhook.serving.knative.dev
   timeoutSeconds: 10
+  rules:
+  - apiGroups:
+    - "*.knative.dev"
+    apiVersions:
+    - v1alpha1
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    scope: "*"
+    resources:
+    - metrics
+    - metrics/status
```

Comment:
I think we  can drop the `/status` subresource here since we can't apply defaults to a resource's `Status`